### ISSUE TITLE
[BUG] Improve protected member usage

### ISSUE THREAD

**Issue by CoMPaTech** [plugwise/python-plugwise#72]
**Describe the bug**
Lots of Access to a protected member _(...)_thread of a client class

**Evidence**
1. `pylint plugwise/*py` (and/or subdirectories)
2. `W0212: Access to a protected member _(...)_thread of a client class (protected-access)`
3. Rewrite these to non-underscore to share outside of class

**Additional context**
Found as part of codefactor improvements and refactoring.
Should be handled with care since we don't have the appropriate tests for the stick part.

**Comment by CoMPaTech** [comment]
List of todo's (usb related, excluding already marked as fixme/todos)
```
************* Module plugwise.stick
plugwise/stick.py:67:0: C0103: Class name "stick" doesn't conform to PascalCase naming style (invalid-name)
plugwise/stick.py:560:15: W0212: Access to a protected member _run_receive_timeout_thread of a client class (protected-access)
plugwise/stick.py:561:23: E1101: Instance of 'Thread' has no 'isAlive' member (no-member)
plugwise/stick.py:561:23: E1101: Instance of 'bool' has no 'isAlive' member (no-member)
plugwise/stick.py:561:23: W0212: Access to a protected member _receive_timeout_thread of a client class (protected-access)
plugwise/stick.py:565:20: W0212: Access to a protected member _receive_timeout_thread of a client class (protected-access)
plugwise/stick.py:567:24: W0212: Access to a protected member _receive_timeout_loop of a client class (protected-access)
plugwise/stick.py:572:20: W0212: Access to a protected member _receive_timeout_thread of a client class (protected-access)
plugwise/stick.py:573:20: W0212: Access to a protected member _receive_timeout_thread of a client class (protected-access)
plugwise/stick.py:575:15: W0212: Access to a protected member _run_send_message_thread of a client class (protected-access)
plugwise/stick.py:576:23: E1101: Instance of 'Thread' has no 'isAlive' member (no-member)
plugwise/stick.py:576:23: W0212: Access to a protected member _send_message_thread of a client class (protected-access)
plugwise/stick.py:580:20: W0212: Access to a protected member _send_message_thread of a client class (protected-access)
plugwise/stick.py:582:24: W0212: Access to a protected member _send_message_loop of a client class (protected-access)
plugwise/stick.py:587:20: W0212: Access to a protected member _send_message_thread of a client class (protected-access)
plugwise/stick.py:588:20: W0212: Access to a protected member _send_message_thread of a client class (protected-access)
plugwise/stick.py:591:23: E1101: Instance of 'Thread' has no 'isAlive' member (no-member)
plugwise/stick.py:635:28: W0212: Access to a protected member _request_ping of a client class (protected-access)
plugwise/stick.py:639:28: W0212: Access to a protected member _request_power_update of a client class (protected-access)
************* Module plugwise.util
plugwise/util.py:156:4: W0231: __init__ method from base class 'BaseType' is not called (super-init-not-called)
************* Module plugwise.connections
plugwise/connections/__init__.py:36:12: E1101: Instance of 'StickConnection' has no '_open_connection' member (no-member)
plugwise/connections/__init__.py:52:19: E1101: Instance of 'StickConnection' has no '_read_data' member (no-member)
plugwise/connections/__init__.py:83:16: E1101: Instance of 'StickConnection' has no '_write_data' member (no-member)
plugwise/connections/__init__.py:103:15: E1101: Instance of 'Thread' has no 'isAlive' member (no-member)
plugwise/connections/__init__.py:107:15: E1101: Instance of 'Thread' has no 'isAlive' member (no-member)
plugwise/connections/__init__.py:120:18: E1101: Instance of 'Thread' has no 'isAlive' member (no-member)
plugwise/connections/__init__.py:123:12: E1101: Instance of 'StickConnection' has no '_close_connection' member (no-member)
************* Module plugwise.messages
plugwise/messages/__init__.py:18:14: E1101: Instance of 'PlugwiseMessage' has no 'ID' member (no-member)
************* Module plugwise.messages.requests
plugwise/messages/requests.py:252:8: C0103: Variable name "t" doesn't conform to snake_case naming style (invalid-name)
plugwise/messages/requests.py:254:8: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
************* Module plugwise.messages.responses
plugwise/messages/responses.py:78:14: E1101: Instance of 'NodeResponse' has no 'ID' member (no-member)
plugwise/messages/responses.py:258:8: C0103: Attribute name "in_RSSI" doesn't conform to snake_case naming style (invalid-name)
plugwise/messages/responses.py:259:8: C0103: Attribute name "out_RSSI" doesn't conform to snake_case naming style (invalid-name)
plugwise/messages/responses.py:340:8: C0103: Attribute name "hz" doesn't conform to snake_case naming style (invalid-name)
************* Module plugwise.nodes
plugwise/nodes/__init__.py:46:8: C0103: Attribute name "_RSSI_in" doesn't conform to snake_case naming style (invalid-name)
plugwise/nodes/__init__.py:47:8: C0103: Attribute name "_RSSI_out" doesn't conform to snake_case naming style (invalid-name)
plugwise/nodes/__init__.py:323:15: E1101: Instance of 'PlugwiseNode' has no '_categories' member (no-member)
plugwise/nodes/__init__.py:330:15: E1101: Instance of 'PlugwiseNode' has no '_sensors' member (no-member)
plugwise/nodes/__init__.py:337:15: E1101: Instance of 'PlugwiseNode' has no '_switches' member (no-member)
plugwise/nodes/__init__.py:365:15: E1101: Instance of 'PlugwiseNode' has no 'self' member (no-member)
plugwise/nodes/__init__.py:392:4: C0103: Method name "get_in_RSSI" doesn't conform to snake_case naming style (invalid-name)
plugwise/nodes/__init__.py:401:4: C0103: Method name "get_out_RSSI" doesn't conform to snake_case naming style (invalid-name)
************* Module plugwise.nodes.circle_plus
plugwise/nodes/circle_plus.py:100:8: C0103: Variable name "dt" doesn't conform to snake_case naming style (invalid-name)
************* Module plugwise.nodes.scan
plugwise/nodes/scan.py:100:4: C0103: Method name "CalibrateLight" doesn't conform to snake_case naming style (invalid-name)
plugwise/nodes/scan.py:104:4: C0103: Method name "Configure_scan" doesn't conform to snake_case naming style (invalid-name)
plugwise/nodes/scan.py:129:4: C0103: Method name "SetMotionAction" doesn't conform to snake_case naming style (invalid-name)
************* Module plugwise.nodes.sed
plugwise/nodes/sed.py:38:8: C0103: Attribute name "_SED_requests" doesn't conform to snake_case naming style (invalid-name)
plugwise/nodes/sed.py:118:4: W0221: Parameters differ from overridden '_request_ping' method (arguments-differ)
plugwise/nodes/sed.py:140:4: C0103: Method name "Configure_SED" doesn't conform to snake_case naming style (invalid-name)
```

**Comment by bouwew** [comment]
#78 and #90 have solved almost all issues. 
This is the present result:
![image](https://user-images.githubusercontent.com/11290930/124639809-e2fdbe80-de8c-11eb-9d4f-f7d8ae06366a.png)

**Comment by bouwew** [comment]
Looks like the removed BaseType-inherit causes new pylint warnings.

**Comment by bouwew** [comment]
The remaining issues were fixed in #91.
